loklak.php

```diff
@@ -55,5 +55,42 @@ public function status() {
 		return json_encode($request, true);
 	}
 
-
+	public function search($query, $since=null, $until=null, $from_user=null, $count=null){
+        $this->requestURL = $this->baseUrl . '/api/search.json';
+        $this->query = $query;
+        $this->since = $since;
+        $this->until = $until;
+        $this->from_user = $from_user;
+        $this->count = $count;
+        if($query){
+            $params = array('q'=>$this->query);
+            // $params['query'] = $this->$query;
+            // if ($since)
+         //        $params['query'] = $params['query'] . ' since:'.$this->since;
+         //    if ($until)
+         //        $params['query'] = $params['query'] . ' until:'.$this->until;
+         //    if ($from_user)
+         //        $params['query'] = $params['query'] . ' from:'.$this->from_user;
+         //    if ($count)
+         //        $params['count'] = $this->count;
+            //print_r($params);
```

Comment:
Construct the other parts also, this only takes a base query, the functionality for
- since
- until
- from_user
- count

are commented out, please push them into the associative array